classes/Policies.php

```diff
@@ -234,12 +250,19 @@ class Policies {
      */
     public function getArray() {
         $out = array();
-        foreach ($this->policies as $policy_id => $policy_text)
-        {
+        if ( $this->policy_id ) {
```

Comment:
If the constructor altered `$this->policies` if policy_id was set, the code here and in limitToSet would be unnecessary, I think?